app/src/main/java/com/example/pinterbest/viewmodels/HomeViewModel.kt

```diff
@@ -1,10 +1,11 @@
 package com.example.pinterbest.viewmodels
 
+import android.content.Context
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.asLiveData
 import com.example.pinterbest.data.repository.Repository
 
-class HomeViewModel(repository: Repository) :
+class HomeViewModel(context: Context, repository: Repository) :
```

Comment:
ViewModels should not to know about context, it's better to keep them without it to make them testable and cleaner

Reply:
Fixed here https://github.com/enginire4ek/Pinterbest/pull/3/commits/f47063b96f0a24a56ad6def81896bbe4dee63167